.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+
+**Neos and Flow is community software built on open-source principles by a community collaborating on a shared goal. Everyone is welcome to contribute to the community and has the possibility to influence the project.**
+
+Please be considerate of our [code of conduct][1]! We like to provide a positive and enjoyable environment for everybody.
+
+Do you want to get a certain improvement in Neos or Flow ? Did you find a problem? Do you want to help translate, promote or document the products?
+
+Become part of the Neos Project Community and help us offer a better experience to the content management industry.
```

Comment:
... and help us building a content management system with a great user experience!